apps/print/frontend/src/components/toolbar.tsx

```diff
@@ -147,29 +159,61 @@ function PrinterStatus({ status }: { status: PrinterStatusType }) {
   return <PrinterConnectionStatus connected={connected} />;
 }
 
-function UsbStatus({ status }: { status: UsbDriveStatus }) {
+function BatteryStatus({ status }: { status?: BatteryInfo }) {
+  return (
+    <Row style={{ gap: '0.25rem', alignItems: 'center' }}>
+      {getBatteryIcon(status, true)}
+      {status && !status.discharging && (
+        <Icons.Bolt style={{ fontSize: '0.8em' }} color="inverse" />
+      )}
+      {status && format.percent(status.level)}
+      {status && status.level < 0.25 && status.discharging && (
+        <Icons.Warning color="inverseWarning" />
+      )}
+    </Row>
+  );
+}
+
+type ExtendedUsbDriveStatus = UsbDriveStatus['status'] | 'ejecting';
+const buttonIconAndText: Record<ExtendedUsbDriveStatus, [IconName, string]> = {
+  no_drive: ['Disabled', 'No USB'],
+  error: ['Disabled', 'No USB'],
+  mounted: ['Eject', 'Eject USB'],
+  ejecting: ['Eject', 'Ejecting...'],
+  ejected: ['Disabled', 'No USB'],
```

Comment:
This `ejected` string should be `USB Ejected` rather than `No USB`. To be clear to the user why the inserted USB drive is not usable. And to match behavior of the other `usbcontroller` button.